src/pages/first-byte-page/fb-home-page/Task.css

```diff
@@ -19,14 +19,48 @@
 
 .task-name {
     text-align: left;
+    max-width: 60%;
 }
 
 .task-time {
     text-align: right;
+    width: 40%;
 }
 
 .task-desc {
     font-size: 1rem;
     color: #6a6a6a;
 
-}
\ No newline at end of file
+}
+
+@media (min-width: 768px) {
+    .task-row{
+        font-size: 1rem;
```

Comment:
instead of setting font-size to a rem value, we can set the html body text size to a pixel value, where the rem can then reflect. 

By default rem: 16px

But we can do html{
font-size: 14px
}


to set rem to be 14px. Unless this was purposefully done for just font size